site/description.html

```diff
@@ -1,3 +1,10 @@
+<p class="notice">
+    Are you at Mathfest this week?  Join us on Friday 8/9 for these four TBIL-themed talks!
+    - "Implementations of Team-Based Inquiry Learning Across Instructional Contexts", 8:40 AM, Room 314.  
+    - "Team-Based Inquiry Learning in College Algebra and Precalculus", 9:20 AM, Room 311. 
+    - "Tailoring TBIL Resources Using Github", 4:40 PM, Room 313. 
+    - "Team-Based Inquiry Learning in Linear Algebra", 5:20 PM, Room 314. 

```

Comment:
Oh, I guess this is HTML but the others are markdown...
```suggestion
<ul>
    <li> "Implementations of Team-Based Inquiry Learning Across Instructional Contexts", 8:40 AM, Room 314. </li>
    <li> "Team-Based Inquiry Learning in College Algebra and Precalculus", 9:20 AM, Room 311. </li>
    <li>"Tailoring TBIL Resources Using Github", 4:40 PM, Room 313. </li>
    <li>"Team-Based Inquiry Learning in Linear Algebra", 5:20 PM, Room 314. </li>
</ul>
```